<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagForm</class>
 <widget class="QWidget" name="TagForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tags</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblTags">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>321</width>
     <height>351</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Tag</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
